changelog.txt

```diff
@@ -0,0 +1,2 @@
+fixed - Fixed the issue that the providerName of generic OAuth, OIDC and SAML providers not defaults to providerId.
```

Comment:
Fixed the issue where the providerName for generic OAuth, OIDC and SAML providers does not default to providerId.